js/cedar.js

```diff
@@ -69,8 +65,8 @@ Cedar.Application = function(options) {
   Cedar.config.wait = this.options.wait;
   Cedar.config.fetch = this.options.fetch;
 
-  if (typeof Cedar.$ === "undefined") {
-    Cedar.$ = jQuery({});
+  if (Cedar.events === undefined) {
```

Comment:
so if i understand correctly, the argument for the long-winded approach using `typeof` is because the js var `undefined` could thoeretically be redefined... which seems pretty unlikely. your approach here is easier to read if slightly less safe. :+1: